### PR TITLE
Legislative Process Page Sidebar

### PR DESCRIPTION
**Pull request by thasarito**
Accidentally clode the previous one

related issue: #28

### CODE REVIEW

**Comment by thasarito** [comment]
Fixed what was mentioned in #33 @Th1nkK1D 

Long time no see indeed, missed this community kub

File: src/components/LegislativeProcess/Sidebar.svelte [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+		}
+	];
+</script>
+
+<SideNav isOpen={true} fixed={false} class="relative z-0">
+	<SideNavItems>
+		{#each sections as section}
+			<SideNavMenu text={section.menu} expanded class="font-bold text-black">
+				{#each section.menuItem as item}
+					<SideNavMenuItem href={`#${item.text}`}>{item.text}</SideNavMenuItem>
```
Comment (Th1nkK1D) [inline src/components/LegislativeProcess/Sidebar.svelte:42]: In Svelte, You can add variable brackets in a string like
```svelte
<SideNavMenuItem href="#{item.text}">{item.text}</SideNavMenuItem>
```

**Review by Th1nkK1D — CHANGES_REQUESTED** [review]
Found a strange bug on a small screenshot.
- An overlayed element appears which dims the content below it.
- When I click on it, it disappears and the sidebar moves to a strange location.

[Screencast from 2023-10-19 12-14-04.webm](https://github.com/wevisdemo/parliament-watch/assets/8860448/964e4dc1-ebfe-4461-be7d-509128cff3e7)

I just merged one PR that fixed some issues with the Navbar, maybe you can try to pull from the `main` branch to see if the issue is still there.

One more point, Regarding the small screen size, I understand that the designer wants the sidebar to have the same width as the content below.
image

Now we have a lot of horizontal margins and some overflow at the bottom.

image

PS. I tested on the latest Firefox and Chromium on Linux. Not sure if you can reproduce this on your computer.

File: src/components/LegislativeProcess/Sidebar.svelte [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			]
+		},
+		{
+			menu: 'ประเภทการลงมติ',
+			menuItem: [{ text: 'ประเภทของมติ' }, { text: 'ผลของการลงมติ' }, { text: 'การลงมติอื่นๆ' }]
+		},
+		{
+			menu: 'ส.ส. และ ส.ว. ทำอะไรบ้างในสภา',
+			menuItem: [
+				{ text: 'หน้าที่ขออง ส.ส.' },
```
Comment (Icyscools) [inline src/components/LegislativeProcess/Sidebar.svelte:29]:
```suggestion
				{ text: 'หน้าที่ของ ส.ส.' },
```

**Review by Icyscools — CHANGES_REQUESTED** [review]
I just want to know how to implement this sidebar and found this typo error. 😅

**Comment by thasarito** [comment]
fixed @Th1nkK1D

**Review by Th1nkK1D — APPROVED** [review]
LGTM! Thank you for your contribution. We will contact you at the end of this month about the sticker!

**Comment by Th1nkK1D** [comment]
Thank you for the Hacktoberfest participation! Please fill up the [form](https://airtable.com/appgGYIo458vDkrQN/shrZMe2hZBa6tQXvz) before 6 November 2023 to receive our limited-edition stickers.

**Pull request merged**